### PR TITLE
Add new workflow Find_InsightConnect_Resources_from_Slack

### PR DESCRIPTION
**Pull request by nhurd-r7**
## Requirements

Workflow contributors, verify you have completed the following items by checking them off:

### Checklist
- [yes] Workflow verified as working in a test
- [yes] Workflow bundle includes `Name_of_Workflow.icon` file, `help.md` file, `workflow.spec.yaml` file, and `screenshots` directory
- [yes] Workflow title should follow general [Title of Articles](http://grammar.yourdictionary.com/capitalization/rules-for-capitalization-in-titles.html) format
- [yes] Workflow includes a detailed Description
- [yes] Workflow step names follow [Title of Articles](http://grammar.yourdictionary.com/capitalization/rules-for-capitalization-in-titles.html) format
- [yes] Triggers, Decisions, and other workflow steps include descriptions where needed
- [no; not necessary for this use case] Workflow includes at least one `Final Report` artifact that summarizes workflow execution details
- [yes] Workflow bundle (directory) is named after the title of the workflow in snake case with first letters capitalized, e.g. `workflows/Name_of_Workflow/`
- [yes] `help.md` file includes Description, Key Features, Requirements, Setup, Technical Details, Version History, and References
- [yes] `workflow.spec.yaml` file includes `products`, `name`, `title`, `description`, `use_cases` tags, `keywords` tags, `source_url`, and the `name` for each screenshot matches a PNG image in the `screenshots` directory

### Assessment

Post your results with screenshots in this PR using the following style:

### Screenshots

Workflow
Slack_loop
icon-start
icon-help
Getting_started_response

#### Workflow Builder

Workflow

#### Job

- No Job screenshot because this Workflow only leverages the Slack app, so I did not feel that the Job would provide any helpful information

#### Artifact

- Similar to the above thought process, this Workflow does not include an Artifact step because it only leverages the Slack app, and was designed to provide the user a simplified 'you don't have to leave Slack' experience. I did include screenshots of the Slack interactions above, which I believe are more helpful in this case.

### CODE REVIEW

File: workflows/Find_InsightConnect_Resources_from_Slack/workflow.spec.yaml [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+name: Find_InsightConnect_Resources_from_Slack
+title: "Find InsightConnect Resources from Slack"
+description: Find InsightConnect resources to support your initial implementation, without ever leaving Slack! Start using this workflow in minutes, without the need for an orchestrator.
+version: 1.0.0
+vendor: rapid7
+support: rapid7
+status: []
+hub_tags:
+  use_cases: [Chat, Slack]
+  keywords: [Chat, Slack]
```
Comment (ncramer-r7) [inline workflows/Find_InsightConnect_Resources_from_Slack/workflow.spec.yaml:12]:
```suggestion
  keywords: [chat, slack]
```

File: workflows/Find_InsightConnect_Resources_from_Slack/workflow.spec.yaml [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+products: ["insightconnect"]
+name: Find_InsightConnect_Resources_from_Slack
+title: "Find InsightConnect Resources from Slack"
+description: Find InsightConnect resources to support your initial implementation, without ever leaving Slack! Start using this workflow in minutes, without the need for an orchestrator.
+version: 1.0.0
+vendor: rapid7
+support: rapid7
+status: []
+hub_tags:
+  use_cases: [Chat, Slack]
```
Comment (ncramer-r7) [inline workflows/Find_InsightConnect_Resources_from_Slack/workflow.spec.yaml:11]:
```suggestion
  use_cases: []
```
We typically only use a set of use cases found [here](https://docs.google.com/document/d/1Z2wV5Z96jtxdz-LVsLzdl3S1aFo9RhVPODmylyJmay0) under the Workflow Tags --> Use Cases section, but I would say this workflow doesn't fit into any of those buckets

File: workflows/Find_InsightConnect_Resources_from_Slack/help.md [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+To run the workflow, send a direct message to your InsightConnect Slack app or @ the app in a channel starting with the command `icon-start`.
+
+For example, in a direct message with the InsightConnect app:
+
+* `icon-start`
+
+Or, in a channel including the app:
+
+* `@Rapid7 InsightConnect icon-start`
+
```
Comment (ncramer-r7) [inline workflows/Find_InsightConnect_Resources_from_Slack/help.md:36]:
```suggestion
_This workflow will trigger in any direct messages to the InsightConnect app, or any message in a channel directed @ the InsightConnect app. The app must first be invited to the channel in order to trigger the workflow this way._

To run the workflow, send a direct message to your InsightConnect Slack app or @ the app in a channel starting with the command `icon-start`.

For example, in a direct message with the InsightConnect app:

* `icon-start`

Or, in a channel including the app:

* `@Rapid7 InsightConnect icon-start`

```
You may want to include some instructions about `icon-help` in here as well

Comment (ncramer-r7) [inline workflows/Find_InsightConnect_Resources_from_Slack/help.md:36]: You may want to include some instruction on `icon-help` in here as well. Ex:

```
To seek further help on your InsightConnect journey, send a direct message to your InsightConnect Slack app or @ the app in a channel starting with the command `icon-help`. 

Follow along with the interactive Slack buttons to find the resources best sorted to your needs.
```

File: workflows/Find_InsightConnect_Resources_from_Slack/Find_InsightConnect_Resources_from_Slack.icon [hunk 4]
```diff
@@ -0,0 +1,1631 @@
...
+          "ac38a0ae-633c-4fec-97cb-3847d3133d8b": {
+            "nodeId": "ac38a0ae-633c-4fec-97cb-3847d3133d8b",
+            "name": "Slack Trigger",
+            "description": "@Rapid7 InsightConnect icon-start; or\n@Rapid7 InsightConnect icon-help",
+            "type": "trigger",
+            "continueOnFailure": false,
+            "isDisabled": false,
+            "parameters": {
+              "input": {
+                "matchChannel": "",
```
Comment (ncramer-r7) [inline workflows/Find_InsightConnect_Resources_from_Slack/Find_InsightConnect_Resources_from_Slack.icon:707]:
```suggestion
                "matchChannel": "change_me",
```
We typically include this "change_me" input to notify the user to change the channel to fit their environment

File: workflows/Find_InsightConnect_Resources_from_Slack/Find_InsightConnect_Resources_from_Slack.icon [hunk 5]
```diff
@@ -0,0 +1,1631 @@
...
+        "humanCostDisplayUnit": "minutes"
+      }
+    ],
+    "triggers": [
+      {
+        "id": "8a646081-bfe4-40e5-829c-dea9f0d00e19",
+        "name": "Slack Trigger",
+        "description": "@Rapid7 InsightConnect icon-start; or\n@Rapid7 InsightConnect icon-help",
+        "input": {
+          "matchChannel": "",
```
Comment (ncramer-r7) [inline workflows/Find_InsightConnect_Resources_from_Slack/Find_InsightConnect_Resources_from_Slack.icon:1372]:
```suggestion
                "matchChannel": "change_me",
```
Same comment as above

**Review by ncramer-r7 — CHANGES_REQUESTED** [review]
I love this workflow! Should help massively with the ICON onboarding experience. The workflow itself looks good to me, tried it out on my side and it is super sleek! I commented on a few things that we do for standardization, but other than that, looks great!

**Review by ncramer-r7 — APPROVED** [review]

**Review by mhofert-r7 — APPROVED** [review]

**Pull request merged**